components/tidb_query/src/codec/mysql/decimal.rs

```diff
@@ -1547,6 +1547,16 @@ impl Decimal {
         Res::Ok(x)
     }
 
+    // TODO, add test
```

Comment:
So, corresponding tests are in other pr?

Reply:
No, I will add test for it latter in this PR.